Tests/NIOHTTP1Tests/HTTPServerUpgradeTests.swift

```diff
@@ -14,7 +14,7 @@
 
 import XCTest
 import Dispatch
-import NIOCore
+@testable import NIOCore
```

Comment:
We could also copy & paste `UnsafeTransfer` and `UnsafeMutableTransferBox` into the test target if we don't want to add `@testable`.

Reply:
Done in https://github.com/apple/swift-nio/pull/2214/commits/8472a2dd92301148daf017eb464ede8ec48f569e